WcaOnRails/config/locales/en.yml

```diff
@@ -384,6 +385,7 @@ en:
         competitor_limit_reason: "What is the reason for the limit on competitors? Please fill this out in English!"
         venueAddress: ""
         championship_type: ""
+        registration_requirements: "Please indicate here any specifities about the registration process, as needed per <a href=\"https://www.worldcubeassociation.org/regulations/guidelines.html#2c1+\" target=\"_blank\">Guideline 2c1+</a>. After the competition is announced, this field will <b>not</b> be editable anymore, and Board's approval will be needed to modify it.<br/>For example indicate here if competitors need to have a clean payment history with a specific entity, or if they will have to answer a confirmation email in case the maximum number of registrations is reached."
```

Comment:
@thewca/wrc-team for review on the wording.

Reply:
Looks good!

Reply:
Is "specifities" a word?

Also, just checking, but does html work here?

Reply:
In Frenglish it definitely is, but it looks like it [does exist](http://www.wordreference.com/enfr/specificity) in English, and is a synonym of "peculiarities".

Although it does mean what I wanted, I'll just switch this to "all details" if it sounds unusual.

Reply:
Oh wow, I can't even read properly my own word, and obviously meant "specificities" :p

Reply:
Hahaha, Frenglish. I think "all details" or "any details" sounds more natural to me =)

Reply:
Oh, or "specificities"! That's a fine word.

Reply:
Yep, I just fixed the typo.